pkg/yurtctl/cmd/revert/revert.go

```diff
@@ -327,6 +342,116 @@ func removeYurtTunnelServer(client *kubernetes.Clientset) error {
 	return nil
 }
 
+func removeYurtAppManager(client *kubernetes.Clientset, yurtAppManagerClientSet dynamic.Interface) error {
+	// 1. remove the Deployment
+	if err := client.AppsV1().
+		Deployments(constants.YurttunnelNamespace).
```

Comment:
How about rename `constants.YurttunnelNamespace` bacause `constants.YurttunnelNamespace` is used in func removeYurtAppManager and func removeYurtTunnelServer.